### ISSUE TITLE
Uncaught Error: (regl) invalid texture shape

### ISSUE THREAD

**Issue by siberiankhatrus** [ojack/atom-hydra#5]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.31.1 x64
**Electron**: 2.0.7
**OS**: Mac OS X 10.13.6
**Thrown From**: [atom-hydra](https://github.com/ojack/atom-hydra) package 0.2.8

### Stack Trace

Uncaught Error: (regl) invalid texture shape

```
At /Users/krystameredith/.atom/packages/atom-hydra/node_modules/regl/dist/regl.js:49

Error: (regl) invalid texture shape
    at raise (/packages/atom-hydra/node_modules/regl/dist/regl.js:47:15)
    at check (/packages/atom-hydra/node_modules/regl/dist/regl.js:54:5)
    at /packages/atom-hydra/node_modules/regl/dist/regl.js:500:3)
    at reglTexture2D (/packages/atom-hydra/node_modules/regl/dist/regl.js:3406:15)
    at /packages/atom-hydra/node_modules/regl/dist/regl.js:3530:5)
    at HydraSource.tick (/packages/atom-hydra/node_modules/hydra-synth/src/hydra-source.js:70:29)
    at HydraSynth.tick (/packages/atom-hydra/node_modules/hydra-synth/index.js:295:17)
    at /packages/atom-hydra/lib/main.js:216:20)
    at emitOne (events.js:116:13)
    at Engine.emit (events.js:211:7)
    at Engine.tick (/packages/atom-hydra/node_modules/raf-loop/index.js:42:10)
```

### Commands

```
     -1:49.9.0 atom-hydra:toggle (input.hidden-input)
     -1:27.2.0 atom-hydra:evalLine (input.hidden-input)
     -1:05.3.0 core:select-all (input.hidden-input)
  2x -1:04.8.0 atom-hydra:evalLine (input.hidden-input)
     -0:32.1.0 core:select-all (input.hidden-input)
     -0:31.8.0 atom-hydra:evalLine (input.hidden-input)
  2x -0:24.7.0 core:select-all (div.tool-panel.tree-view)
     -0:21.9.0 atom-hydra:evalLine (input.hidden-input)
```

### Non-Core Packages

```
atom-hydra 0.2.8 
```

**Comment by ojack** [comment]
what code were you running?

**Comment by eerieear** [comment]
Hi Olivia , 

I got a similar error running the example code 
"5-video"
I only replaced the path to the file :

vid.src = atom.project.getPaths()[0]+'/Volumes/DATA/e_e/IMAGEN/MOSHED-2018-8-12-12-49-55.webm'

I tried .gif also and got the same error
Console only shows:
(regl) invalid texture shape

The message also shows with the original code :

vid.src = atom.project.getPaths()[0]+'/assets/jelly.webm'

where jelly.webm is local to the examples project

Any help is much appreciated
Thanks for making this great environment
Cheers !

e_e

**Comment by zachkrall** [comment]
Hi @eerieear 

When you have 5-video.js open, is the hydra-examples directory also open in the Project sidebar on the left? `atom.project.getPaths()[0]` takes the current atom project directory path. 

You can also use your absolute path as is like below:
```
vid.src = '/Volumes/DATA/e_e/IMAGEN/MOSHED-2018-8-12-12-49-55.webm'
```

**Comment by ojack** [comment]
Closing this as @zachkrall's answer should fix it for you. This error happens when a texture is not properly loaded.

**Comment by eerieear** [comment]
Hi Guys ! very late feedback on this , sorry I did not follow up. I worked around it and never checked again , thank you @zachkrall and @ojack for all the work you are doing. I had a successful first ever Live Visuals night on Saturday after using Hydra for about 2 months It worked fine even recording my screen for many hours , I had no errors and a great overall performance, here is a video of the last hour , there is some delay but I can say it worked very good during the performance. @zachkrall I am now studying your workshop cheatsheet for next time. https://youtu.be/N7pvDoIeeWI

Again Thank both you so much for all the attention and love you put into this.

Cheers 
Seb
